lib/isaac.go

```diff
@@ -120,7 +113,6 @@ func (is *ISAAC) receiveBallotStateINIT(ballot Ballot) (vs VotingStateStaging, e
 		// self-sign
 		newBallot.SetState(sebakcommon.BallotStateINIT)
 		newBallot.Vote(VotingYES) // The BallotStateINIT ballot will have 'VotingYES'
-		newBallot.UpdateHash()
 		newBallot.Sign(is.Node.Keypair(), is.networkID)
```

Comment:
ballot.UpdateHash() doesn't need to be called in here cause it is called in ballot.Sign() method.